jekyll/_cci2/skip-build.md

```diff
@@ -1,27 +1,35 @@
 ---
 layout: classic-docs
-title: Skip and Cancel Builds
-short-title: Skip and Cancel Builds
-description: How to Prevent CircleCI From Automatically Building
+title: Skip or Cancel Pipelines
+description: This document describes how to prevent automatically building your project.
```

Comment:
Are we happy to use "building", but not "build"?

Reply:
This is hard!! something like:
"This document describes the options available to you for controlling when work is automatically carried out on your project, by skipping jobs or auto-cancelling workflows"

Reply:
I will also add some use case info about preventing unnecessary credit usage.

Reply:
Sounds good. I think your alternative wording there is fine.